core/data/fetch.py

```diff
@@ -11,12 +11,13 @@
 
 from core.util.logging.logable import Logable
 from core.util.util import timing, setup_log, log_ent_exit
-from core.util.constants import IMGDIR_PATH, MERGE_COLS, BFLY_FAMILY, BFLY_LIFESTAGE, DATASET_PATH, DIRNAME_DELIM
+from core.util.constants import IMGDIR_PATH, MERGE_COLS, BFLY_FAMILY, BFLY_LIFESTAGE, DATASET_PATH, DIRNAME_DELIM, RAW_WORLD_DATA_PATH, RAW_WORLD_LABEL_PATH
 from core.data.feature import FeatureExtractor
 
 
 class Database(Logable):
     def __init__(self,
+                 min_amount_of_pictures,
```

Comment:
Didn't we agree to also include number of species? For experimental sake, we could see the effect on model precision, if we include more images. We dont have unlimited GRAM.